src/modules/endpoint/__tests__/DefaultDataContainer.spec.js

```diff
@@ -115,11 +115,8 @@ describe('DefaultDataContainer', () => {
     expect(receivedTestEvents).toEqual([testEvents[0]])
     expect(receivedAnotherTestEvents).toEqual([testEvents[1]])
   })
-  // cache is not equal to testResources
-  // (lastUpdate is a moment object in the expected data, but a string in the received data)
-  // will be fixed in NATIVE-330
-  // eslint-disable-next-line jest/no-disabled-tests
-  it.skip('should return the resources associated with the context', async () => {
+
+  it('should return the resources associated with the context', async () => {
```

Comment:
All good, but can you fix the spelling in line 130?
`should return an empty object if no resources *were* found`
:D

Reply:
Sure, only for you :P